search/includes/classes/queue/class-cron.php

```diff
@@ -30,6 +30,16 @@ class Cron {
 	 */
 	const SWEEPER_CRON_INTERVAL = 5 * \MINUTE_IN_SECONDS;
 
+	/**
+	 * The name of the cron event for processing term updates
+	 */
+	const TERM_UPDATE_CRON_EVENT_NAME = 'vip_search_term_update';
+
+	/**
+	 * The number of objects queued per batch for term updates
+	 */
+	const TERM_UPDATE_BATCH_SIZE = 25000;
```

Comment:
I think this batch size is too high...something in the 1k - 5k range would be safer.

Reply:
I've used it in excess of 100k for the speed gain. 1-5k took over an hour to add all the posts to the queue. It went down to less than 20 minutes by increasing it super high.

It feels almost like each page has a lengthy sleep at the end of it with how it was working. The batch size shouldn't have that much of an effect with no other changes.